src/api/app/assets/javascripts/webui/nav_tabs.js

```diff
@@ -26,10 +26,13 @@ $(document).ready(function () {
     setCollapsible(); // jshint ignore:line
   });
 
-  function changeActionHashOfButtonTo() {
+  function replaceURLActionHash() {
     var buttonToAction = $('.button_to').attr('action');
     if (typeof buttonToAction !== 'undefined') {
       $('.button_to').attr('action', buttonToAction.replace(/#.+/, '') + document.location.hash);
     }
+    $('.activity-link').each(function() {
+      this.href += document.location.hash;
+    })
```

Comment:
```suggestion
    });
```

Linters' failing because of this

Reply:
Fixed. :+1: